lively.ide/studio/controls/border.cp.js

```diff
@@ -75,6 +75,12 @@ export class BorderControlModel extends PropertySectionModel {
         }
       },
 
+      expose: {
+        get () {
+          return ['onMouseDown'];
+        }
+      },
+
```

Comment:
Better to work with a binding here, since exposing will override the default onMouseDown of this view entirely (without calling super).